util/SERIALQRUN

```diff
@@ -1,10 +1,10 @@
 sudo qemu-system-x86_64 -s -cpu Opteron_G1 -smp 8 -m 2048  \
```

Comment:
The point of this script was to work on serial console only machines. You've removed the serial argument, left the nographic argument in place, and added via arguments ... this is no longer a serial port invocation. I'm wondering why the change is needed.